CenterForOpenScience/osf.io
# [ENG-1550] Admin app institutional group management.

## Purpose

The admin app will need the functionality to manage the new institutional permission groups so some institutional users can view the new metrics.

## Changes

Adding a new view, form, and html template to support this functionality.

## QA Notes


    - Is this an additive or subtractive change, other?
Additive change
  - How can QA verify? (Through UI, API, AdminApp or AdminAdminApp?)
This might need to be devtested as we need to ensure users are added successfully to permission groups

## Documentation

N/A

## Side Effects

There shouldn't be 

## Ticket

https://openscience.atlassian.net/browse/ENG-1550
